types/fulu/execution_payload.yaml

```diff
@@ -0,0 +1,81 @@
+Fulu:
+  ExecutionPayloadCommon:
+    # An abstract object to collect the common fields between the ExecutionPayload and the ExecutionPayloadHeader objects.
+    type: object
+    # TODO: update link once confirmed.
+    description: "The [`ExecutionPayload`](https://github.com/ethereum/consensus-specs/blob/v1.5.0-beta.2/specs/fulu/beacon-chain.md#executionpayload) object from the CL Fulu spec."
+    required:
+      [
+        parent_hash,
+        fee_recipient,
+        state_root,
+        receipts_root,
+        logs_bloom,
+        prev_randao,
+        block_number,
+        gas_limit,
+        gas_used,
+        timestamp,
+        extra_data,
+        base_fee_per_gas,
+        blob_gas_used,
+        excess_blob_gas,
+        block_hash,
+      ]
+    properties:
+      parent_hash:
+        $ref: "../primitive.yaml#/Root"
+      fee_recipient:
+        $ref: "../primitive.yaml#/ExecutionAddress"
+      state_root:
+        $ref: "../primitive.yaml#/Root"
+      receipts_root:
+        $ref: "../primitive.yaml#/Root"
+      logs_bloom:
+        $ref: "../primitive.yaml#/LogsBloom"
+      prev_randao:
+        $ref: "../primitive.yaml#/Bytes32"
+      block_number:
+        $ref: "../primitive.yaml#/Uint64"
+      gas_limit:
+        $ref: "../primitive.yaml#/Uint64"
+      gas_used:
+        $ref: "../primitive.yaml#/Uint64"
+      timestamp:
+        $ref: "../primitive.yaml#/Uint64"
+      extra_data:
+        $ref: "../primitive.yaml#/ExtraData"
+      base_fee_per_gas:
+        $ref: "../primitive.yaml#/Uint256"
+      blob_gas_used:
+        $ref: "../primitive.yaml#/Uint256"
+      excess_blob_gas:
+        $ref: "../primitive.yaml#/Uint256"
+      block_hash:
+        $ref: "../primitive.yaml#/Root"
+
+  ExecutionPayload:
+    allOf:
+      - $ref: "#/Fulu/ExecutionPayloadCommon"
+      - type: object
+        required: [transactions, withdrawals, proof_version]
+        properties:
+          transactions:
+            $ref: "../bellatrix/transactions.yaml#/Bellatrix/Transactions"
+          withdrawals:
+            $ref: "../capella/withdrawals.yaml#/Capella/Withdrawals"
+          proof_version:
```

Comment:
is this the only field that actually changes compared to Electra which requires to update `BeaconBlockBody`? I can't find that in the CL spec

Reply:
Yes, this is the only field change, also cc @jtraglia to see if we want to add that to CL spec

Reply:
>  also cc @jtraglia to see if we want to add that to CL spec

that would be good otherwise I am not sure how to merge this as beacon-api generally follows the consensus spec, but we can merge this soon, just wanna do another minor release for Electra before that in a few days

Reply:
Ah sorry just now seeing this. I will make a PR for this.

Reply:
@0x00101010 why do we have the proof version here?

the proof version byte is just to disambiguate the EL wrapper, i don't see where it leaks into the CL at all

Reply:
I've removed Fulu BeaconBlock / SignedBeaconBlock / ExecutionPayload, since it does not change, we can reuse the Electra definition